src/main.ts

```diff
@@ -1,14 +1,15 @@
-import { Logger as NestLogger } from '@nestjs/common';
 import { NestApplication, NestFactory } from '@nestjs/core';
 import { MainModule } from '@ukef/main.module';
 
 import { App } from './app';
+import { REDACT_STRINGS } from './constants';
+import { ConsoleLoggerWithRedact } from './logging/console-logger-with-redact';
 
 const main = async () => {
-  const nestApp: NestApplication = await NestFactory.create(MainModule, { bufferLogs: true });
+  const logger = new ConsoleLoggerWithRedact(REDACT_STRINGS);
+  const nestApp: NestApplication = await NestFactory.create(MainModule, { logger, bufferLogs: false });
   const app = new App(nestApp);
 
-  const logger = new NestLogger();
```

Comment:
Do we lose out on much functionality by replacing the `NestLogger` with the `ConsoleLoggerWithRedact`?